contentcuration/contentcuration/frontend/channelEdit/vuex/contentNode/getters.js

```diff
@@ -78,6 +78,35 @@ export function getTopicAndResourceCounts(state) {
   };
 }
 
+export function getIsImported(state) {
+  return function(id) {
+    const node = getContentNode(state)(id);
+
+    return !!(
+      node &&
+      node.original_source_node_id &&
+      node.node_id !== node.original_source_node_id
+    );
+  };
+}
+
+export function getImportedChannelLink(state) {
+  return function(id) {
+    const node = getContentNode(state)(id);
+    if (node && getIsImported(state)(id)) {
+      return `/channels/${node.original_channel_id}/#/originalSourceNode/${node.original_source_node_id}`;
```

Comment:
This is brittle - we should form this using a combination of server side URLs:
`window.Urls.channel(node.original_channel_id)` and the router reversal URL for the route, using `router.resolve`: https://router.vuejs.org/api/#router-resolve